app/src/debug/java/com/artemzin/qualitymatters/ui/presenters/DeveloperSettingsPresenter.java

```diff
@@ -23,15 +23,22 @@ public DeveloperSettingsPresenter(@NonNull DeveloperSettingsModelImpl developerS
     @Override
     public void bindView(@NonNull DeveloperSettingsView view) {
         super.bindView(view);
+        syncDeveloperSettings();
+    }
+
+    public void syncDeveloperSettings() {
+        DeveloperSettingsView view = view();
 
-        view.changeGitSha(developerSettingsModel.getGitSha());
-        view.changeBuildDate(developerSettingsModel.getBuildDate());
-        view.changeBuildVersionCode(developerSettingsModel.getBuildVersionCode());
-        view.changeBuildVersionName(developerSettingsModel.getBuildVersionName());
-        view.changeStethoState(developerSettingsModel.isStethoEnabled());
-        view.changeLeakCanaryState(developerSettingsModel.isLeakCanaryEnabled());
-        view.changeTinyDancerState(developerSettingsModel.isTinyDancerEnabled());
-        view.changeHttpLoggingLevel(developerSettingsModel.getHttpLoggingLevel());
+        if (view != null) {
```

Comment:
I would actually crash here instead of no-op, we shouldn't call this method when `view == null`